pal/src/configure.cmake

```diff
@@ -1,3 +1,5 @@
+if (NOT CC_TARGET_OS_ANDROID)
```

Comment:
PAL uses this file to feature detect on target. This is not an option when cross-compiling the code. 

> can you make a note to clean up this file at some point?

Good idea, will do. Thanks